.github/workflows/go.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         run: |
           git config --global user.email you@example.com
           git config --global user.name Your Name
-          make all
-          make test-docker
+          IN_CI=true make all
```

Comment:
Does `make all` need it?

Reply:
I think you're right that `make all` doesn't need it, will remove